sdk/core/System.Net.ClientModel/tests/client/OpenAIClient/Choice.Serialization.cs

```diff
@@ -74,7 +79,17 @@ internal static Choice DeserializeChoice(JsonElement element)
         internal static Choice FromResponse(PipelineResponse response)
         {
             using var document = JsonDocument.Parse(response.Content);
```

Comment:
nit: do we want to stop using JsonDocument.Parse entirely in favor of Utf8JsonReader per performance cost?

Reply:
Yes, but writing Utf8JsonReader code by hand is super hard and error prone. I know that Micheal wants to generate it and then we would remove all the JsonDocument.Parse calls.